grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/tracing/TracingServerInterceptor.java

```diff
@@ -0,0 +1,52 @@
+package org.lognet.springboot.grpc.tracing;
+
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanBuilder;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.api.trace.TraceFlags;
+import io.opentelemetry.api.trace.TraceState;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+
+public class TracingServerInterceptor implements ServerInterceptor {
```

Comment:
Should have the highest  precedence https://github.com/LogNet/grpc-spring-boot-starter#421-interceptors-ordering

Reply:
Do you mean we need to add such info to the doc?
Like `Use @Order(10) with TracingServerInterceptor `?

Reply:
added @Order(20)? I don't see you use `@Order`. Have you had in mind any other mechanisms?

Reply:
see [this](https://github.com/LogNet/grpc-spring-boot-starter#421-interceptors-ordering:~:text=and%20Metrics%20integration.-,Their%20order%20can%20also%20be%20controlled%20by%20below%20properties%20%3A,-grpc.recovery.interceptor), we will need to move `recovery` and  `auth` down `1`

Reply:
I've set HIGHEST_PRECEDENCE. Do I need to touch recovery and auth here? Please, let me know if you want to change order somehow